rfd/0188/README.md

```diff
@@ -0,0 +1,204 @@
+---
+authors: Nahum Shalman <nshalman@edgecast.io>
+state: draft
+discussion: https://github.com/TritonDataCenter/rfd/issues?q=%22RFD+188%22
+---
+
+# RFD 188 Manatee CMON Monitoring Integration
+
+## Problem Statement
+
+We currently operate multiple Manatee clusters across different environments
+(Triton itself and Manta) but lack automated monitoring and alerting for
+PostgreSQL cluster health status.
+
+[Manatee](https://github.com/TritonDataCenter/manatee/blob/master/docs/user-guide.md#what-is-manatee)
+is the replicated PostgreSQL cluster that powers TritonDataCenter.
+If a Manatee cluster cannot accept writes due to a degraded cluster state, the
+entire TritonDC becomes non-operational until the cluster is restored to a
+healthy state.
+
+If a Manatee cluster cannot accept writes because of a degraded cluster state,
+its TritonDC is not operational until the Manatee cluster has been restored to a
+healthy state. We already have alerting in place for Manatee instance disk usage
+getting high, which is a common source of issues.
+
+The missing piece for monitoring Manatee is alerting based on cluster health
+status, commonly viewed by entering a terminal, connecting to a Manatee instance
```

Comment:
Split this into two sentences. One suggestion:  "... alerting based on cluster health status.  Today that status is viewable by the `manatee-adm pg-status` command entered in the manatee zone's shell on the Triton head node."

Reply:
Copied verbatim from an internal document. But I will clean it up for this RFD. Thanks!